docs/reference/getting-started.asciidoc

```diff
@@ -288,7 +288,13 @@ epoch      timestamp cluster       status node.total node.data shards pri relo i
 
 We can see that our cluster named "elasticsearch" is up with a green status.
 
-Whenever we ask for the cluster health, we either get green, yellow, or red. Green means everything is good (cluster is fully functional), yellow means all data is available but some replicas are not yet allocated (cluster is fully functional), and red means some data is not available for whatever reason. Note that even if a cluster is red, it still is partially functional (i.e. it will continue to serve search requests from the available shards) but you will likely need to fix it ASAP since you have missing data.
+Whenever we ask for the cluster health, we either get green, yellow, or red.
+
+    * Green - everything is good (cluster is fully functional)
+    * Yellow - all data is available but some replicas are not yet allocated (cluster is fully functional)
+    * Red - some data is not available for whatever reason (cluster is partially functional)
+
+**Note:** When a cluster is red, it will continue to serve search requests from the available shards but you will likely need to fix it ASAP since you have missing data.
```

Comment:
I would shy away from using the term "missing data" in this sentence, instead, I think "unassigned shards" would be a better phrase.

Reply:
seems logical. Will make the changes. 👍

Reply:
@dakrone I have made the changes.